docs/single-spa-config.md

```diff
@@ -0,0 +1,95 @@
+# single spa config
+The single-spa single spa config consists of all code that is not part of a
+[registered application](/docs/applications.md). Ideally, this only includes an html file
+and a javascript file that declares single-spa registered applications. It is best practice to keep your
+single spa config as small as possible and to simply defer to single-spa to manage
+all of the registered applications. The single spa config should not be doing client-side html
+rendering nor should it be responding to routing events such as `hashchange` or `popstate`.
+Instead, all of that functionality should be taken care of either by single-spa itself or by
+a registered application.
+
+## Index.html file
+The main thing that you should be doing in your html file is executing your single spa config. For your
+use case, this could mean something like `<script src="/single-spa-config.js"></script>`.
+Example:
+```js
+<html>
+  <body>
+    <script src="/single-spa-config.js"></script>
+  </body>
+</html>
+```
+
+## Declaring registered applications
+
+Declaring a [registered application](/docs/applications.md) is registering it with single-spa so that it will be loaded,
```

Comment:
> Declaring a registered application is registering it

->

> You must register applications with single-spa so that it nows how and when to load and mount them.